index.js

```diff
@@ -1182,20 +1170,24 @@ exports.main = function() {
         console.error("Could not reload snippet content: " + e);
     }
 
-    // Check whether more than 24 hours have elsapsed since the last time a sidebar was shown
-    // Also, make sure that the user has not actually completed the on-boarding process.
-    if ((getTimeElapsed(simpleStorage.lastSidebarLaunchTime) >= defaultSidebarInterval || 
-        ((timeElapsedFormula*(defaultSidebarInterval - (getTimeElapsed(simpleStorage.lastSidebarLaunchTime)))) < 60000)) 
-        && typeof simpleStorage.rewardSidebarShown === 'undefined' )  {
+    // catch the anomaly where users aren't getting a notification because their lastSidebarLaunchTime isn't set, but they ARE onboarding
+    if(typeof simpleStorage.lastSidebarLaunchTime === 'undefined' && simpleStorage.isOnBoarding === true) {
```

Comment:
This is to catch the users who are currently within the flow. We are setting their lastSidebarLaunchTime to 24 hours in the past, which means they will hit the next conditional an they will have a 60 second notification timer start upon update.